tools/misbehaving-jmx-server/Dockerfile

```diff
@@ -9,22 +12,41 @@ WORKDIR /app
 # Copy the pom.xml and Maven files and install the dependencies
 COPY .mvn .mvn/
 COPY pom.xml mvnw mvnw.cmd ./
+
+# Test containers bug doesn't seem to allow mount cache
+# RUN --mount=type=cache,id=mavenCache,target=/root/.m2,sharing=locked \
 RUN set -eu &&  \
     ./mvnw dependency:resolve;
 
 # Copy the source code and build the JAR file
 COPY src/ src/
+
+# Test containers bug doesn't seem to allow mount cache
+# RUN --mount=type=cache,id=mavenCache,target=/root/.m2,sharing=locked \
```

Comment:
Are these useful to keep in the code? Doesn't seem like it from my perspective

Reply:
I'll change it to a `TODO` (or warning) to investigate why this doesn't work in Test containers. Ideally want this to work so that the Maven cache is shared between all builds of Misbehaving JMX Server even if you change the base or final image.